src/app/settings/emails/emails.component.ts

```diff
@@ -1,20 +1,43 @@
 import { Component, OnInit } from '@angular/core';
-import { Router }            from '@angular/router';
+import { Router } from '@angular/router';
 
+import { ProfileService } from './../../profile/profile.service';
+import { Email } from './../../models/email';
 
 @Component({
   selector: 'alm-settingsOverview',
   templateUrl: 'emails.component.html',
   styleUrls: ['./emails.component.scss']
 })
-export class EmailsComponent implements OnInit {
+export class EmailsComponent {
+
+  email: string;
+  keepPrivate: boolean;
 
   constructor(
-    private router: Router) {
+    private router: Router, public profile: ProfileService) {
+  }
+
+  delete(del: Email): void {
```

Comment:
'delete' is a reserved word.
https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#221-reserved-words